torch/distributed/fsdp/flat_param.py

```diff
@@ -14,9 +14,15 @@
 )
 
 import torch
+import torch.distributed as dist
+import torch.distributed._shard.sharding_spec as shard_spec
 import torch.nn as nn
 import torch.nn.functional as F
 from torch import Tensor
+from torch.distributed._shard.sharded_tensor.api import ShardedTensor
+from spmd.tensor import Tensor as DistributedTensor, DeviceMesh
```

Comment:
this needs to guarded in a try/except so people can use it without pipy installed.

Reply:
sure.

Reply:
Added.